upf/upf.api

```diff
@@ -223,15 +235,19 @@ autoreply define upf_nwi_add_del {
   u8 add;
   u32 ip4_table_id;
   u32 ip6_table_id;
+  u8 ipfix_policy[32];
+  vl_api_address_t ipfix_collector_ip;
   u8 nwi_len;
   u8 nwi[nwi_len];
 };
 
 define upf_nwi_details {
   u32 client_index;
   u32 context;
-  u32 ip4_fib_table;
-  u32 ip6_fib_table;
+  u32 ip4_table_id;
+  u32 ip6_table_id;
+  u8 ipfix_policy[32];
```

Comment:
limiting the policy name to 31 characters is a bit strange.

Reply:
Extended it to 64 bytes, perhaps that's should be enough for now :)
Presently, the set of policies is fixed (only `none`, `dest` and `default`)